### PR TITLE
[WIP] Fix `code-blocks-checker`

### PR DESCRIPTION
**Pull request by HeahDude**
This PR is in sync with https://github.com/symfony-tools/code-block-checker/pull/44.

### CODE REVIEW

File: create_framework/http_kernel_controller_resolver.rst [hunk 1]
```diff
@@ -103,29 +103,29 @@ The ``index()`` method needs the Request object as an argument.
 ``getArguments()`` knows when to inject it properly if it is type-hinted
 correctly::
 
-    public function index(Request $request)
+    public function index(Request $request) { ... }
```
Comment (OskarStark) [inline create_framework/http_kernel_controller_resolver.rst:106]: Imho it is too verbose just to make the code checker happy. What about allowing it somehow as is?

Comment (wouterj) [reply create_framework/http_kernel_controller_resolver.rst:106]: I agree. In the past (when this tool was introduced), we also decided that it's not the goal of documentation examples to be 100% syntax correct. Their goal is to explain the PHP API and do as good of a job as possible on not distracting the user with irrelevant details.

I'm -1 for these additions.

**Comment by Nyholm** [comment]
If we rerun the CI some errors should be fixed

**Comment by javiereguiluz** [comment]
Jules, I must close this proposal without merging because at least two members of the Docs Team are against this change. I'm sorry, but I thank you for this proposal and for the fixes you did in symfony-tools. Thanks!

**Pull request closed without merging**